test/hook_line_and_slacker_test.exs

```diff
@@ -0,0 +1,72 @@
+defmodule HookLineAndSlackerTest do
+  use ExUnit.Case, async: true
+  use Plug.Test
+  doctest HookLineAndSlacker
+
+  @opts HookLineAndSlacker.WebhookListener.init([])
+
+  import Mock
```

Comment:
Mocking is an antipattern according to José Valim [here](http://blog.plataformatec.com.br/2015/10/mocks-and-explicit-contracts/). I did it like this for speed of implementation.

A better pattern would be mock as a *noun* rather than mock as a *verb* and actually build a mock `Slackex.Chat` object for testing.